README.md

```diff
@@ -1,48 +1,109 @@
+## Integrating with rake
 
-You'll now have access to some new rake tasks:
+Create a rake task with the desired configuration. For example, this will create a rake task called `i18n:hygiene`:
 
-| Task | Description |
-|---|---|
-| `i18n:hygiene:all` | Runs all of the checks |
-| `i18n:hygiene:check_key_usage` | Reports any translations that are unused |
-| `i18n:hygiene:check_variables` | Reports any translations that have an incorrectly named or missing interpolated variables |
-| `i18n:hygiene:check_entities` | Reports any translations that have unexpected HTML entities |
-| `i18n:hygiene:check_return_symbols` | Reports any translations that have a unicode return character in them |
-| `i18n:hygiene:check_script_tags` | Reports any translations that have script tags in them |
+```ruby
+namespace :i18n do
+  I18n::Hygiene::RakeTask.new do |config|
+    config.directories = ["app", "lib"]
+  end
+end
 
-#### Without Rails
+```
+
+You can then run the rake task with:
+```
+bundle exec rake i18n:hygiene
+```
 
-Using this gem without Rails is intended to be possible, but it isn't very resilient to bare configurations yet. We hope to improve that.
+You could also create separate rake tasks with different names and configurations, this may be useful if you are in the middle of rolling out a new locale:
+```ruby
+namespace :i18n do
+  I18n::Hygiene::RakeTask.new(:hygiene_live) do |config|
+    config.locales = [:fr]
+  end
 
-You can still give it a try, you'll need to include this in your projects Rakefile:
+  I18n::Hygiene::RakeTask.new(:hygiene_wip) do |config|
+    config.locales = [:es]
+  end
+end
+```
 
+Which could be run like:
+
+```
+bundle exec rake i18n:hygiene_live
+bundle exec rake i18n:hygiene_wip
 ```
-require 'i18n/hygiene'
-spec = Gem::Specification.find_by_name 'i18n-hygiene'
-load "#{spec.gem_dir}/lib/tasks/i18n_hygiene.rake"
+
+## Configuration
+
+| Configuration | Default | Description |
+|---|---|---|
+| `concurrency` | Number of CPU cores | How many threads to use for key usage check |
+| `directories` | All | Directories to search for key usage |
+| `exclude_files` | None | Excludes files from key usage check |
+| `file_extensions` | `rb, erb, coffee, js, jsx` | Only look in files with these extensions for key usage |
+| `primary_locale` | `I18n.default_locale` | Translations from other locales are checked against this |
+| `locales` | `I18n.available_locales` | Translations from these are checked against primary |
+| `keys_to_exclude` | None | Exclude individual keys  |
+| `scopes_to_exclude` | None | Exclude groups of keys |
+
+Example using all configuration options:
+
+```ruby
+I18n::Hygiene::RakeTask.new do |config|
+  config.concurrency = 16
+  config.directories = ["app", "lib"]
+  config.exclude_files = ["README.md"]
+  config.file_extensions = ["rb", "jsx"]
+  config.primary_locale = :en
+  config.locales = [:ja, :kr]
+  config.keys_to_exclude = [
```

Comment:
I think it would be nice to call this `exclude_keys` and `exclude_scopes`, like `exclude_files`. Obviously this doesn't have to be part of the 1.0 release.